src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
@danielbdias @Sauloxd @albertossilva I thought we can keep just toJson() and have some method to return a object by context ex.
     
    const user = new User({ email, password });
    return user.getContext('astronaut').toJSON()